doc/getstarted/README.md

```diff
@@ -0,0 +1,110 @@
+# Get started 
+
+It's recommended to begin by exploring the 👉 **[basic terms](../design/architecture#1-basic-terms)** 👈 to better understand the documentation.
+
+## Simple deploy
+
+```
+docker run --network host emcmongoose/mongoose-base
+```
+
+☝️ This command will launch mongoose in **standalone mode**, and the **storage mock** will be used as the tested [storage](../design/architecture#1-basic-terms).
```

Comment:
Not sure that there is a need in this link. The definition of storage in basic terms is that "storage that's tested", so tested storage here pretty much covers the definition. Just my 2 cents